pkg/adapter/rule.go

```diff
@@ -34,7 +35,8 @@ const (
 // Source: 192.168.1.1/nginx:latest
 // Target: 192.168.1.1/nginx:latest-suffix
 func addSuffix(ref, suffix string) (string, error) {
-	named, err := docker.ParseDockerRef(ref)
+	re := regexp.MustCompile(`@sha256.*`)
```

Comment:
Can we write a test case for this? It converts the digested reference format into `repo:latest`?
Maybe better to handle this in `if _, ok := named.(docker.Digested); ok {`.

Reply:
Add test case in `rule_test.go`.